app/utils/dictionary/payment.js

```diff
@@ -31,6 +34,7 @@ export const paymentCurrencies = [
     symbol : 'R$',
     name   : 'Brazilian real',
     stripe : true,
+    alipay : true,
```

Comment:
@mrsaicharan1 AliPay doesn't support all the currencies you have added `true` to , Please take this as reference and make following changes
<img width="786" alt="Screenshot 2019-05-29 at 4 59 08 AM" src="https://user-images.githubusercontent.com/44091822/58518865-84a28400-81ce-11e9-8d07-601566ed3611.png">

Reply:
@kushthedude Yeah, I just checked it out. It was using Stripe Sources , but I checked it out later.

Reply:
Cool , Make appropriate changes 😄